packages/chord-mark/src/renderer/components/renderLine.js

```diff
@@ -30,9 +38,13 @@ export default function render(
 	if (isFromSectionMultiply) {
 		lineClasses.push('cmLine--isFromSectionMultiply');
 	}
-
+	
 	return lineTpl({
 		line,
 		lineClasses: lineClasses.join(' '),
+		shouldOpenSection: shouldOpenSection,
+		sectionWrapperClasses: sectionWrapperClasses.join(' '),
+		shouldClosePriorSection: shouldClosePriorSection,
+		shouldCloseFinalSection: shouldCloseFinalSection,
```

Comment:
You can use object shorthand declaration here as well:

```javascript
{
		line,
		lineClasses: lineClasses.join(' '),
		shouldOpenSection,
		sectionWrapperClasses: sectionWrapperClasses.join(' '),
		shouldClosePriorSection,
		shouldCloseFinalSection,
}
```
https://javascript.plainenglish.io/object-literals-using-object-property-shorthand-6360825c60ef

Reply:
Good catch, yep.